src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int upperBase;
+    private final int lowerBase;
+    private final int height;
+
+    public IsoscelesTrapezoid(int upperBase, int lowerBase, int height, String color) {
+        super(color);
+        this.upperBase = upperBase;
+        this.lowerBase = lowerBase;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) ((upperBase + lowerBase) * height) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: "
```

Comment:
If you're using java 15+, you can use java text blocks (""" ... """)

Reply:
I don't understand how to use java text blocks in this case if I need System.out.println() to output everything in one line, with getArea(), getColor() and other variables added